webapp/src/main/webapp/i18n/all_de_DE.properties

```
@@ -0,0 +1,910 @@
+imageUpload.errorUnknown = Leider waren wir nicht in der Lage, das bereitgestellte Foto zu verarbeiten. Bitte versuchen Sie es mit einem anderen Foto.
+imageUpload.errorFileTooBig = Bitte laden Sie ein Bild kleiner als {0} Megabyte hoch.
+imageUpload.errorUnrecognizedFileType = '' {0} '' ist kein akzeptierter Bilddateityp. Bitte laden Sie JPEG, GIF oder PNG-Dateien hoch.
+imageUpload.errorNoPhotoSelected = Bitte wählen Sie ein Foto aus.
+imageUpload.errorBadMultipartRequest = Parsen der Multi-Part-Anfrage für den Bild-Upload fehlgeschlagen.
+imageUpload.errorFormFieldMissing = Das Formular enthielt kein '' {0} '' Feld. "
+					
+#
+# User Accounts pages
+#
+account_management = Kontenverwaltung
+user_accounts_link = Benutzerkonten
+user_accounts_title = Benutzerkonten
+
+login_count = Login-Zähler
+last_login = Letzte Anmeldung
+
+add_new_account = Neues Konto hinzufügen
+edit_account = Konto bearbeiten
+external_auth_only = Nur extern Authentifizierte
+reset_password = Passwort zurücksetzen
+reset_password_note = Hinweis: Anweisungen zum Zurücksetzen des Passworts werden an die oben angegebenen Adresse gesendet. Das Passwort wird nicht zurückgesetzt, bis der Benutzer den Link in dieser E-Mail geöffnet hat.
+new_password = Neues Passwort
+confirm_password = Neues Passwort bestätigen
+minimum_password_length = Mindestens {0} Zeichen, höchstens {1}.
+leave_password_unchanged = Wenn Sie dieses Feld leer lassen, wird das Passwort nicht verändert.
+confirm_initial_password = Initiales Passwort bestätigen
+
+new_account_1 = Ein neues Konto für
+new_account_2 = wurde erfolgreich erstellt.
+new_account_title = Neues Konto
+new_account_notification = An {0} wurde eine Benachrichtigungs-E-Mail mit einer Anleitung zur Konten-Aktivierung und zur Erstellung eines Passwort gesendet.
+updated_account_1 = Das Konto für
+updated_account_2 = wurde aktualisiert.
+updated_account_title = Aktualisiertes Konto
+updated_account_notification = An {0} wurde eine Bestätigungs-E-Mail mit einer Anleitung zum Zurücksetzen des Passwortes gesendet. Das Passwort wird nicht zurückgesetzt, bis der Benutzer den Link in dieser E-Mail geöffnet hat.
+deleted_accounts = {0} {0, Auswahl, 0#Konten|1#Konto|1<Konten} gelöscht .
+
+enter_new_password = Bitte geben Sie Ihr neues Passwort für {0} ein.
+error_no_email_address = Bitte geben Sie Ihre E-Mail-Adresse ein.
+error_no_password = Bitte geben Sie Ihr Passwort ein.
+error_incorrect_credentials = Die E-Mail oder das Passwort ist falsch.
+logins_disabled_for_maintenance = Benutzer-Logins sind vorübergehend deaktiviert, während das System gewartet wird.
+error_no_new_password = Bitte geben Sie ihr neues Passwort ein.
+error_passwords_dont_match = Die eingegebenen Passwörter stimmen nicht überein.
+error_password_length = Bitte geben Sie ein Passwort zwischen {0} und {1} Zeichen Länge ein.
+error_previous_password = Ihr neues Passwort darf nicht dem Aktuellen entsprechen.
+
```

```suggestion
password_system_has_changed = Our password system has been upgraded to make it more secure. \
Please enter a new password.

```
Please add this string, it is now part of Vitro all.properties. See https://github.com/vivo-project/Vitro/commit/fea33a456a8755efebaaf4737526a1487ade3608#diff-4d548d21dbf1c2f5a307a6f2a1ce1882R126. Would be best if it was added in German :)

---

Added a translation for password migration.